src/Dropdown.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useMemo, useState } from 'react';
+import { Person } from './types/Person';
+import cn from 'classnames';
+import debounce from 'lodash.debounce';
+
+type Props = {
+  delay: number;
+  people: Person[];
+  onSelected: (person: Person | null) => void;
+};
+
+export const Dropdown: React.FC<Props> = ({ delay, people, onSelected }) => {
+  const [inputValue, setInputValue] = useState('');
+  const [visible, setVisible] = useState(false);
+
+  const debounceName = useMemo(
+    () => debounce(() => setVisible(true), delay),
+    [delay],
+  );
+
+  const filteredPeople = useMemo(() => {
```

Comment:
Not sure that you really want to use useMemo here, caching here may only decrease the performance.

Reply:
without memo it do not work as I expect ;)